### PR TITLE
use dotenv to seperate config data from code.

### PR DESCRIPTION
**Pull request by strand**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,4 +41,4 @@ bower_components/
 temp
 
 # Configuration constants files
-/cfg/*.const.js
\ No newline at end of file
+/cfg/*.const.js
```
Comment (strand) [inline .gitignore]: Remove this line and any cfg/*const files

**Comment by maggiepint** [comment]
No issues with the pull, but I need to try to point the CI build at this and make sure everything works.

**Comment by strand** [comment]
Have we configured the CI build?

**Comment by strand** [comment]
@macrobug I'd still like to get this stale work in… is the CI build setup now?

**Comment by strand** [comment]
screen shot 2018-02-26 at 10 26 04 am

**Pull request closed without merging**